app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
+      def index
+        @messages = Message.all
```

Comment:
- [ ] Great effort working on your project, but currently, On the front end i realize the random greeting is not rendered on the screen. I would kindly suggest you work on it and be sure the greetings are showing on the screen
Below is a video demonstration

Video 👇 

[screen-capture (1).webm](https://github.com/hasnatali1947/Hello_backEnd/assets/78634841/a44a57a5-b8fe-40f0-b062-98ecf8b8e647)

Reply:
well resolved! ✔️